public/editor/scripts/editor/js/fc/filesystem-sync.js

```diff
@@ -36,8 +36,8 @@ define(function(require) {
 
     // Update the UI with a "Saving..." indicator whenever we sync a file
     syncManager.on("file-sync-start", function() {
+	  $("#navbar-save-indicator").removeClass("hide");
       $("#navbar-save-indicator").text("{{ fileSavingIndicator }}");
```

Comment:
don't know why my editor is messing up with my indentation! it looks perfect om my machine, but whenever I add it pushes line 39 back!